zebra-chain/src/utils.rs

```diff
@@ -0,0 +1,15 @@
+//! Utility functions to be used in several crates.
```

Comment:
This comment is now outdated
```suggestion
//! Utility functions for chain data.
```